generators/messaging-adapter-generator/src/test/java/uk/gov/justice/subscription/jms/parser/EventSourcesFileParserTest.java

```diff
@@ -88,7 +58,7 @@ public void shouldThrowExceptionIfNoEventSourcesPresent() {
         try {
             eventSourcesFileParser.getEventSources(baseDir, emptyList());
             fail();
-        } catch (final SubscriptionFileParserException e) {
+        } catch (final Exception e) {
```

Comment:
No assertion that the correct Exception is thrown

Reply:
done